src/gpu/intel/compute/device_info.hpp

```diff
@@ -280,6 +280,7 @@ struct device_info_t {
     int gpu_product_family_ = 0;
     int stepping_id_ = 0;
     uint32_t ip_version_ = 0;
+    bool is_integrated_ = false;
```

Comment:
After the changes in https://github.com/uxlfoundation/oneDNN/pull/3428, which adds `ngen::Product` to the `device_info_t`,  we can just directly query Product, rather than maintaining this extra field.